pkg/api/values.go

```diff
@@ -19,6 +20,7 @@ const (
 	CosignImage      = "COSIGN_IMAGE"
 	RegistryImage    = "REGISTRY_IMAGE"
 	TsaURL           = "TSA_URL"
+	HeadlessUI       = "HEADLESSUI"
```

Comment:
Lets use the same naming convention as other ENV variables

```suggestion
	HeadlessUI       = "HEADLESS_UI"
```